terraform_shared/secretsmanager.tf

```
@@ -2,4 +2,8 @@ resource "aws_secretsmanager_secret" "notify_api_key" {
   name = "notify-api-key"
 
   tags   = "${local.default_tags}"
+}
+
+data "aws_secretsmanager_secret_version" "notify_api_key" {
+  secret_id = "${aws_secretsmanager_secret.notify_api_key.id}"
```

We're now reading this from a data source rather than the resource

```suggestion
  secret_id = "${data.aws_secretsmanager_secret.notify_api_key.id}"
```